cmd/lint.go

```diff
@@ -312,6 +316,23 @@ func lint(args []string, params *lintCommandParams) (report.Report, error) {
 		m.Timer(regalmetrics.RegalConfigParse).Stop()
 	}
 
+	go func() {
+		dir := ""
+		if regalDir != nil {
+			dir = regalDir.Name()
+		}
+
+		if userConfig.Features.RemoteFeatures.CheckVersion &&
```

Comment:
Config loading is handled in a few places, lint, fix and the lsp, in each place, there are minor differences in when and how the config is loaded. Depending on config for this version check functionality makes it more tightly coupled, and harder to just run anywhere. So not sure, about the config stuff in this PR, but we can remove if we don't like it and just offer the env var.